lib/config.h.in

```diff
@@ -72,8 +72,9 @@
 
 #define rm_log_perrorf(message, ...)                                                     \
     if(errno) {{                                                                         \
+        int _errsv = errno;                                                              \
```

Comment:
I guess you have to explain that to me. Does ``g_strdup_printf()`` set ``errno``?